src/core/utils/misc.cpp

```diff
@@ -226,13 +226,30 @@ QPoint Utils::Misc::screenCenter(QWidget *win)
 #endif
 
 /**
- * Detects the version of python by calling
- * "python --version" and parsing the output.
+ * Detects the python version.
  */
 int Utils::Misc::pythonVersion()
 {
     static int version = -1;
     if (version < 0) {
+        QString version_complete = pythonVersionComplete().trimmed();
```

Comment:
I think we are informally transitioning to a style like this `QString versionComplete`. So try to use it too.